1.5.0

```diff
@@ -0,0 +1,42 @@
+{
+    "instructionList": [
+        {
+            "sObjectName": "Account",
+            "instructionTypeName": "command__AC Account (Household) Layout",
+            "taskList": [
+                {
+                    "jiraTicket": "ac-23908a",
+                    "description": "Add Stage field below AccountName in household Layout",
+                    "actionType": "Field",
+                    "actionTypeName": "command__Account_Stage__c",
+                    "optionsMap": {
+                        "fieldExist": "true"
```

Comment:
fieldExist is not a very useful name. After reviewing the code that matches this instruction I think we should use 'doNotModifyExisting' which if true will ignore changes if the field already exists on the page layout. I'll say more on this in the appropriate sf-ac PR